tests/test-cases/Group6-VIC-Machine/6-04-Create-Basic.robot

```diff
@@ -25,107 +22,264 @@ Create VCH - custom base disk
+    ${output}=  Run  bin/vic-machine-linux create --name=${vch-name}-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA --target="%{TEST_USERNAME}:%{TEST_PASSWORD}@%{TEST_URL}" --thumbprint=%{TEST_THUMBPRINT} --image-store=%{TEST_DATASTORE} --bridge-network=%{BRIDGE_NETWORK} --external-network=%{EXTERNAL_NETWORK} ${vicmachinetls}
+    Should Contain  ${output}  exceeds the permitted 31 characters limit
+
+    # Delete the portgroup added by env vars keyword
+    Cleanup VCH Bridge Network  ${vch-name}
+
+Create VCH - Existing VCH name
+    Set Test Environment Variables
+    Run Keyword And Ignore Error  Cleanup Dangling VMs On Test Server
+    Run Keyword And Ignore Error  Cleanup Datastore On Test Server
+
+    ${output}=  Run  bin/vic-machine-linux create --name=${vch-name} --target="%{TEST_USERNAME}:%{TEST_PASSWORD}@%{TEST_URL}" --thumbprint=%{TEST_THUMBPRINT} --image-store=%{TEST_DATASTORE} --bridge-network=%{BRIDGE_NETWORK} --external-network=%{EXTERNAL_NETWORK} ${vicmachinetls}
     Should Contain  ${output}  Installer completed successfully
     Get Docker Params  ${output}  ${true}
-    Log To Console  Installer completed successfully: ${vch-name}...
+    Log To Console  Installer completed successfully: ${vch-name}
+
+    ${output}=  Run  bin/vic-machine-linux create --name=${vch-name} --target="%{TEST_USERNAME}:%{TEST_PASSWORD}@%{TEST_URL}" --thumbprint=%{TEST_THUMBPRINT} --image-store=%{TEST_DATASTORE} --bridge-network=%{BRIDGE_NETWORK} --external-network=%{EXTERNAL_NETWORK} ${vicmachinetls}
+    Should Contain  ${output}  exists, to install with same name, please delete it first
 
-    Run Regression Tests
     Cleanup VIC Appliance On Test Server
 
-Create VCH - full params
-    Log To Console  \nRunning vic-machine create
-    # Attempt to cleanup old/canceled tests
+Create VCH - Existing VM name
+    Set Test Environment Variables
     Run Keyword And Ignore Error  Cleanup Dangling VMs On Test Server
     Run Keyword And Ignore Error  Cleanup Datastore On Test Server
+
+    # Create dummy VM
+    ${rc}  ${output}=  Run And Return Rc And Output  govc vm.create -net=%{EXTERNAL_NETWORK} ${vch-name}
+    Should Be Equal As Integers  ${rc}  0
+
+    ${output}=  Run  bin/vic-machine-linux create --name=${vch-name} --target="%{TEST_USERNAME}:%{TEST_PASSWORD}@%{TEST_URL}" --thumbprint=%{TEST_THUMBPRINT} --image-store=%{TEST_DATASTORE} --bridge-network=%{BRIDGE_NETWORK} --external-network=%{EXTERNAL_NETWORK} ${vicmachinetls}
+    Get Docker Params  ${output}  ${true}
+    Log  ${output}
+    Should Contain  ${output}  Installer completed successfully
+    Log To Console  Installer completed successfully: ${vch-name}
+
+    Run Keyword And Ignore Error  Cleanup VIC Appliance On Test Server
+    ${rc}  ${output}=  Run And Return Rc And Output  govc vm.destroy ${vch-name}
+    Should Be Equal As Integers  ${rc}  0
+    Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Run  govc host.portgroup.remove ${vch-name}-bridge
```

Comment:
that didn't work because the keyword returns an error before it gets to this govc command